app/services/DynamoChannelTestsAudit.scala

```diff
@@ -0,0 +1,54 @@
+package services
+
+import com.typesafe.scalalogging.StrictLogging
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.syntax.EncoderOps
+import io.circe.{Decoder, Encoder}
+import models.ChannelTest
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient
+import software.amazon.awssdk.services.dynamodb.model._
+import zio.{ZEnv, ZIO}
+import models.DynamoErrors._
+import services.DynamoChannelTestsAudit.ChannelTestAudit
+import utils.Circe.jsonToDynamo
+
+import java.time.OffsetDateTime
+
+object DynamoChannelTestsAudit {
+  // The model that we write to the audit table
+  case class ChannelTestAudit[T <: ChannelTest[T] : Encoder : Decoder](
```

Comment:
Here we support the creation of audits for any type of ChannelTest, and it must have a JSON encoder + decoder